### DISCUSSION TITLE
Welcome to gpt-subtrans discussions!

### DISCUSSION THREAD

**Discussion by machinewrapped** [machinewrapped/llm-subtrans#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by LoganMai** [comment]
hi. I encountered an error after running it, can you help me troubleshoot it?"

-error_code=insufficient_quota error_message='You exceeded your current quota, please check your plan and billing details.' error_param=None error_type=insufficient_quota message='OpenAI API error received' stream_error=False
- Rate limit hit, quota exceeded. Please wait until the quota resets.
- Failed to translate scene 1 (You exceeded your current quota, please check your plan and billing details.)... finishing

And I want to translate it into Vietnamese, can you help me set it up?

**Reply by machinewrapped** [reply]
Hi, that's an error from OpenAI, you need to make sure you are using a valid API key:

https://platform.openai.com/account/api-keys

Apparently some people need to use a different base URL to access the OpenAI/GPT service, perhaps because the default is blocked in their country - I'm not sure what people use instead though.

Translating to Vietnamese should be no problem if the app can connect to the GPT service.

**Reply by lynnwan0415** [reply]
Hi. I encountered the same issue. I've changed my API key 3 times but it still can not translate the subtitles. It keeps showing 'Error executing' 'You exceeded your current quota……' Did you fix your error?

**Reply by machinewrapped** [reply]
Are you on a paid plan or free trial? What do you see at https://platform.openai.com/account/usage - is it registering any requests?

I don't believe there's anything I can do on my side, I can only make the request with the API key you provide and see what comes back from OpenAI. 

As far as I'm aware the default API_BASE should work for the vast majority of users (https://api.openai.com/v1) but it's possible it's blocked in your environment.
